ddtrace/ext/aws.py

```diff
@@ -37,9 +38,12 @@ def add_span_arg_tags(
 ):
     # type: (...) -> None
     if endpoint_name not in EXCLUDED_ENDPOINT:
-        tags = dict((name, value) for (name, value) in zip(args_names, args) if name in args_traced)
-        flat_tags = flatten_dict(tags, exclude=EXCLUDED_ENDPOINT_TAGS.get(endpoint_name))
-        span.set_tags({k: truncate_arg_value(v) for k, v in flat_tags.items()})
+        set_flatten_tags(
```

Comment:
With this change, `flatten_dict` becomes dead code internally. Do you reckon it's safe to remove it, @Kyle-Verhoog @brettlangdon ?

Reply:
if it isn't used anywhere else, yeah, I'd say go for it?